prime-dotnet-webapi/Models/EnrolmentCertificate.cs

```diff
@@ -20,9 +20,6 @@ public sealed class EnrolmentCertificate
         public IEnumerable<OrganizationType> OrganizationTypes { get; set; }
 
         public EnrolmentCertificateNote EnrolmentCertificateNote { get; set; }
```

Comment:
this line should also be removed I assume

Reply:
oh yep